src/app/components/feed/FeedFilters.js

```diff
@@ -163,7 +198,7 @@ const FeedFilters = ({
                 filterKey="range"
                 onChange={handleDateRange}
                 value={value || { request_created_at: {} }}
-                optionsToHide={['created_at', 'media_published_at', 'updated_at', 'report_published_at']}
+                optionsToHide={['created_at', 'media_published_at', 'report_published_at', 'request_created_at']}
```

Comment:
Oh this is good, `report_published_at` is so much more descriptive than `updated_at`